simple/utils/spectra.py

```diff
@@ -179,13 +177,17 @@ def ingest_spectrum(
     flags["content"] = row_data
 
     try:
+        # Removing old version of ingesting with ORM, which used schema.py
         # Attempt to add spectrum to database
         # This will throw errors based on validation in schema.py
         # and any database checks (as for example IntegrityError)
-        obj = Spectra(**row_data)
-        with db.session as session:
-            session.add(obj)
-            session.commit()
+        # obj = Spectra(**row_data)
+        # with db.session as session:
+        #     session.add(obj)
+        #     session.commit()
+        with db.engine.connect() as conn:
+            conn.execute(db.Spectra.insert().values(row_data))
+            conn.commit()
```

Comment:
This had to change here (and in Parallaxes and SpectralTypes) since we were using the ORM directly for ingesting and that is no longer available now that schema.py is deprecated. This means we wont' get the SQLAlchemy validation checks, but will still get any DB checks.